### PR TITLE
trivial: Start writing some release notes for 2.0.0

### PR DESCRIPTION
**Pull request by hughsie**
It's not ready for tagging yet, but maybe soon.

### CODE REVIEW

File: data/org.freedesktop.fwupd.metainfo.xml [hunk 1]
```diff
@@ -38,6 +38,414 @@
     <binary>fwupdtool</binary>
   </provides>
   <releases>
+    <release version="2.0.0" date="2024-08-27">
+      <description>
+        <p>
+          This new major release adds the following project-wide features:
+        </p>
```
Comment (superm1) [inline data/org.freedesktop.fwupd.metainfo.xml:45]: need to make noise about the ABI change

Comment (hughsie) [reply data/org.freedesktop.fwupd.metainfo.xml:45]: I've added `Remove deprecated API from libfwupd, bumping soname of libfwupd` further down, but maybe that's not explicit enough. How about:

`This new major release beaks libfwupd ABI and API, to add the following project-wide features`

Comment (superm1) [reply data/org.freedesktop.fwupd.metainfo.xml:45]: Yes.

Comment (hughsie) [reply data/org.freedesktop.fwupd.metainfo.xml:45]: Cool, thanks. Feel free to add fixups to that branch  - i think we should aim to get 2.0.0 by the end of next week or we're never going to get there. I'm under a bit of pressure from various companies to start tagging releases from the main branch again. I think we can make 1.9.x a "bugfixes only" policy then.

Comment (superm1) [reply data/org.freedesktop.fwupd.metainfo.xml]: > i think we should aim to get 2.0.0 by the end of next week or we're never going to get there.

The one thing I would really like to see before tagging 2.0.0 is the emulation capture and use for non-USB working.  I would feel a lot more comfortable with living with all the symbols if we know there are no gaps with that.

Comment (hughsie) [reply data/org.freedesktop.fwupd.metainfo.xml]: Me also; I've got a couple of PRs to do to get that over the line. [ssss](https://github.com/fwupd/fwupd/pull/7501) would be the first that list.

**Comment by hughsie** [comment]
@superm1 I'll merge this (so I can add to it), but don't worry -- no release is imminent :)

**Pull request merged**